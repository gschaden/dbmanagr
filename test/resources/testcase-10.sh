db-navigator.py -s skiline@localhost/ski
