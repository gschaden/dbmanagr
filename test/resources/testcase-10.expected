Id	Title	Subtitle	Autocomplete	
98894356-eb9e-3001-a945-2a0c65d118b4	skiline@localhost/skiline/	Database	skiline@localhost/skiline/
17a8d909-ec18-3fb9-a668-a6b964c1b2b2	skiline@localhost/skiline_test/	Database	skiline@localhost/skiline_test/

